examples/src/main/java/com/adobe/platform/ecosystem/examples/catalog/impl/CatalogServiceImpl.java

```diff
@@ -132,7 +144,7 @@ private String getDataSetProperties() {
      * {@inheritDoc}
      */
     @Override
-    public Batch createBatch(String imsOrg, String authToken, JSONObject payload) throws ConnectorSDKException {
+    public Batch createBatch(String imsOrg, String sandboxName, String authToken, JSONObject payload) throws ConnectorSDKException {
         Batch batch;
```

Comment:
Wont this be a breaking change? already existing integrations will break if this jar is used?
Please confirm.
 If that is the case, my suggestion would be to overload the method as follows:

public Batch createBatch(String imsOrg, String authToken, JSONObject payload) {
this.createBatch(imsOrg, null , authToken, payload);
}
public Batch createBatch(String imsOrg, String sandboxName, String authToken, JSONObject payload) {
}

Reply:
@nids 
This function is not exposed externally and all calls made to this have been checked and either value or null are passed on to ensure this doesn't break. [Existing mappings and new mappings from INFA UI were created and verified as well]
Even if sandboxName is not present it will be **set to null before it reaches** this function.